cran-comments.md

```diff
@@ -8,5 +8,4 @@ This release fixes R CMD check issues in R-devel, as requested by CRAN.
 
 ## Reverse dependencies
 
-* I have run R CMD check on the downstream dependencies, no regressions were
-  found.
+* No regressions were found in R CMD check on the downstream dependencies.
```

Comment:
revert this?

Reply:
I kind of like the passive voice here because it's not clear who "I" is referring to in the active voice version. WDYT?

Reply:
"I" refers to the person submitting to CRAN, could be "We" if we want to include all the maintainers instead.

Reply:
Anyway, it's not clear what the change has to do with the current PR, I'd assumed this was just a "change anything" edit to get CI running.

Reply:
Fair enough; I was commenting in general :) 

Let's revert this change from this PR